server/schedule/placement/rule.go

```diff
@@ -66,7 +66,8 @@ type Rule struct {
 	LocationLabels   []string          `json:"location_labels,omitempty"`   // used to make peers isolated physically
 	IsolationLevel   string            `json:"isolation_level,omitempty"`   // used to isolate replicas explicitly and forcibly
 
-	group *RuleGroup // only set at runtime, no need to {,un}marshal or persist.
+	version uint64     // only set at runtime, add 1 each time rules updated, begin from 0.
```

Comment:
I think we can consider making it persistent and public. Clients may benefit from it to avoid some checks.

Reply:
updated.